httptrace/server.go

```diff
@@ -165,3 +165,9 @@ func (r *responseInfoRecorder) partialResponse() *http.Response {
 		Header:        r.Header(),
 	}
 }
+
+func (r *responseInfoRecorder) Flush() {
```

Comment:
Since this is an exported method, it should have documentation. Source: https://github.com/golang/go/wiki/CodeReviewComments#doc-comments.